Wind direction stats and plots for global wave

QiShi-NOAA opened the pull request:
<b>Note to developers: You must use this PR template!</b>

## Description of Changes

> Please include a summary of the changes and the related GitHub issue(s). Please also include relevant motivation and context.

This PR adds wind direction verification metrics (dir_me, dir_mae and dir_rmse) for global wave. 
## Developer Questions and Checklist
* Is this a high priority PR? If so, why and is there a date it needs to be merged by? No
* Do you have any planned upcoming annual leave/PTO? No
* Are there any changes needed in the times when the jobs are supposed to run/kick-off? Yes, it's need for GFS official verification.
  
- [x] The code changes follow [NCO's EE2 Standards](https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf).
- [x] Developer's name is removed throughout the code and have used `${USER}` where necessary throughout the code.
- [x] References the feature branch for `HOMEevs` are removed from the code.
- [x] J-Job environment variables, COMIN and COMOUT directories, and output follow what has been [defined](https://docs.google.com/document/d/1JWg_4q80aYmmAoD21GFjp9R9y5-3w7WGM3-0HJk0Pjs/edit#heading=h.7ysbr191vzu4) for EVS.
- [x] Jobs over 15 minutes in runtime have restart capability.
- [x] If applicable, changes in the `dev/drivers/scripts` or `dev/modulefiles` have been made in the corresponding `ecf/scripts` and `ecf/defs/evs-nco.def`? 
- [x] Jobs contain the appropriate file checking and don't run METplus for any missing data.
- [x] Code is using METplus wrappers structure and not calling MET executables directly.
- [x] Log is free of any ERRORs or WARNINGs.

## Testing Instructions

> Please include testing instructions for the PR assignee. Include all relevant input datasets needed to run the tests.
Clone my fork https://github.com/QiShi-NOAA/EVS.git
Checkout the branch feature/wind_direction
Link to the fix directory.
Run jevs_global_det_gfs_wave_grid2obs_stats.sh and jevs_global_det_wave_grid2obs_plots_last*.sh to test.
The  archived stats for plotting is located at: /lfs/h2/emc/vpppg/noscrub/qi.shi/EVS_wind_direction/evs/v2.0

Commit:
output u10/v10 from prebuffer for wave wind direction verification

QiShi-NOAA committed:
Merge branch 'NOAA-EMC:develop' into wind_direction

QiShi-NOAA committed:
Merge branch 'NOAA-EMC:develop' into wind_direction

QiShi-NOAA committed:
Merge branch 'NOAA-EMC:develop' into wind_direction

QiShi-NOAA committed:
Merge branch 'NOAA-EMC:develop' into wind_direction

Commit:
modify global-det plot scripts to add wind direction stats for wave
verification

malloryprow commented on ush/global_det/global_det_wave_plots_grid2obs_copy_plots_WindDir.sh:
```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+###############################################################################
+# Name of Script: global_det_wave_copy_plots.sh
+# Developers: Mallory Row / Mallory.Row@noaa.gov
```
You can change my information to your information

malloryprow commented on ush/global_det/global_det_wave_plots_grid2obs_leadaverages_WindDir.sh:
```diff
@@ -0,0 +1,118 @@
+###############################################################################
+# Name of Script: global_det_wave_leadaverages.sh
+# Developers: Deanna Spindler / Deanna.Spindler@noaa.gov
+#             Mallory Row / Mallory.Row@noaa.gov
```
You can change my information to your information

malloryprow replied:
Or you can add you name to the list of developers

malloryprow commented on ush/global_det/global_det_wave_plots_grid2obs_timeseries_WindDir.sh:
```diff
@@ -0,0 +1,121 @@
+###############################################################################
+# Name of Script: global_det_wave_timeseries.sh
+# Developers: Deanna Spindler / Deanna.Spindler@noaa.gov
+#             Mallory Row / Mallory.Row@noaa.gov
```
You can change my information to your information or add or name to the list of developers

Commit:
Add Qi Shi to the list of developers

malloryprow commented:
### 1. jevs_global_det_gfs_wave_grid2obs_stats.sh
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/stats/global_det/jevs_global_det_gfs_wave_grid2obs_stats_00.o181716267
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_gfs_wave_grid2obs_stats_00.181716267.cbqs01
**COMOUT:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/evs/v2.0/stats/global_det

QiShi-NOAA commented:
Hi Mallory,

I've added my names  to the list of developers.

On Wed, Feb 26, 2025 at 10:52 AM Mallory Row ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In ush/global_det/global_det_wave_plots_grid2obs_timeseries_WindDir.sh
> <https://github.com/NOAA-EMC/EVS/pull/671#discussion_r1971866990>:
>
> > @@ -0,0 +1,121 @@
> +#!/bin/bash
> +###############################################################################
> +# Name of Script: global_det_wave_timeseries.sh
> +# Developers: Deanna Spindler / ***@***.***
> +#             Mallory Row / ***@***.***
>
> You can change my information to your information or add or name to the
> list of developers
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#pullrequestreview-2645004875>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN27LMUXPRTTGGZ6ZVLT2RXPKHAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDMNBVGAYDIOBXGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

QiShi-NOAA commented:
The output stats looks good.No ERROR, error, or Error in the log file.
Below is two messages related to the job when I ran check_evs_logs.sh over
the log file:

nid003166: 2025-02-26T16:09:09.912303+00:00 nid003166
systemd-tmpfiles[537152]: /usr/lib/tmpfiles.d/inn.conf:1: Failed to resolve
user 'news': No such process

nid003166: 2025-02-26T16:09:09.915887+00:00 nid003166
systemd-tmpfiles[537152]: /usr/lib/tmpfiles.d/svnserve.conf:1: Failed to
resolve user 'svn': No such process


On Wed, Feb 26, 2025 at 11:07 AM Mallory Row ***@***.***>
wrote:

> 1. jevs_global_det_gfs_wave_grid2obs_stats.sh
>
> *Log File:*
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/stats/global_det/
> *DATA:*
> /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_gfs_wave_grid2obs_stats_00.181716267.cbqs01
> *COMOUT:*
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/evs/v2.0/stats/global_det
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2685518180>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN23G6DGUPDUZWC7XQXT2RXREFAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBVGUYTQMJYGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> [image: malloryprow]*malloryprow* left a comment (NOAA-EMC/EVS#671)
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2685518180>
> 1. jevs_global_det_gfs_wave_grid2obs_stats.sh
>
> *Log File:*
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/stats/global_det/
> *DATA:*
> /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_gfs_wave_grid2obs_stats_00.181716267.cbqs01
> *COMOUT:*
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/evs/v2.0/stats/global_det
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2685518180>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN23G6DGUPDUZWC7XQXT2RXREFAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBVGUYTQMJYGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

malloryprow commented:
I saw that too. I don't think that is anything we can control and is related to things on WCOSS2.

AliciaBentley-NOAA commented on ush/global_det/settings.py:
```diff
@@ -719,7 +719,7 @@ def __init__(self):
                                     'UGRD': 'Zonal Wind Speed',
                                     'VGRD': 'Meridional Wind Speed',
-                                    'UGRD_VGRD': 'Vector Wind Speed',
+                                    'UGRD_VGRD': 'Wind Direction',
```
If 'UGRD_VGRD' has been changed to 'Wind Direction', what is being used for 'Vector Wind Speed' (which is something we would still like to verify out of EVS)? Are the U and V Wind Speed components being used for that? I'd hate to remove something we still want. Thanks!

CC @QiShi-NOAA @malloryprow

AliciaBentley-NOAA replied:
May have answered my own question:
`'UGRD_VGRD10m': 'Surface Vector Wind Speed',`

Is this the replacement?

CC @QiShi-NOAA @malloryprow

AliciaBentley-NOAA reviewed (approved):
I have reviewed this PR and think that it looks very good! I asked one question about continuing to verify vector wind speed once that parameter was changed to 'wind direction', but I scrolled down and believe wind speed was added back later in the same file under a new name. I approve this PR to be merged after successful testing (e.g., proof that wind direction AND wind speed verification looks good). Thanks!

CC @malloryprow @QiShi-NOAA

QiShi-NOAA commented:
So the plot scripts automatically use this definition for the second line
of plot tile and y axis. I can change it back to Vector wind speed and add
if statements in the plot script if that is  preferred.

On Wed, Feb 26, 2025 at 11:46 AM Alicia Bentley ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In ush/global_det/settings.py
> <https://github.com/NOAA-EMC/EVS/pull/671#discussion_r1971963621>:
>
> > @@ -719,7 +719,7 @@ def __init__(self):
>                                      'DPT': 'Dewpoint Temperature',
>                                      'UGRD': 'Zonal Wind Speed',
>                                      'VGRD': 'Meridional Wind Speed',
> -                                    'UGRD_VGRD': 'Vector Wind Speed',
> +                                    'UGRD_VGRD': 'Wind Direction',
>
> If 'UGRD_VGRD' has been changed to 'Wind Direction', what is being used
> for 'Vector Wind Speed' (which is something we would still like to verify
> out of EVS)? Are the U and V Wind Speed components being used for that? I'd
> hate to remove something we still want. Thanks!
>
> CC @QiShi-NOAA <https://github.com/QiShi-NOAA> @malloryprow
> <https://github.com/malloryprow>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#pullrequestreview-2645161986>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN27HIQTXXX57637ECCD2RXVWDAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDMNBVGE3DCOJYGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

malloryprow commented:
For Wind Speed, global_det wave is using the "WIND" variable. 'UGRD_VGRD' is technically Vector Winds but global_det wave isn't using that.

So the changes work, but if statements may be useful to prevent any problems in the future.

QiShi-NOAA commented:
I agree, I'll work on the if statement for plots. It should be easy.

Best,
Qi

On Wed, Feb 26, 2025 at 11:57 AM Mallory Row ***@***.***>
wrote:

> For Wind Speed, global_det wave is using the "WIND" variable. 'UGRD_VGRD'
> is technically Vector Winds but global_det wave isn't using that.
>
> So the changes work, but if statements may be useful to prevent any
> problems in the future.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2685651387>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN23ULEYDYXNCLU7OGET2RXXBHAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBVGY2TCMZYG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: malloryprow]*malloryprow* left a comment (NOAA-EMC/EVS#671)
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2685651387>
>
> For Wind Speed, global_det wave is using the "WIND" variable. 'UGRD_VGRD'
> is technically Vector Winds but global_det wave isn't using that.
>
> So the changes work, but if statements may be useful to prevent any
> problems in the future.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2685651387>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN23ULEYDYXNCLU7OGET2RXXBHAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBVGY2TCMZYG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Commit:
add if statement to change the title2 and ylabel (from vertor wind speed
to wind direction) for wind direction
modify the file name in global_det_wave_plots comments section
stats  plot in time series.py and lead_average.py

Commit:
remove redundant condition

QiShi-NOAA commented:
I've added the if statement in time_series.py and lead_average.py. And I've
restored the UGRD_VGRD variable description to vector wind speed.

On Wed, Feb 26, 2025 at 12:00 PM Qi Shi - NOAA Affiliate ***@***.***>
wrote:

> I agree, I'll work on the if statement for plots. It should be easy.
>
> Best,
> Qi
>
> On Wed, Feb 26, 2025 at 11:57 AM Mallory Row ***@***.***>
> wrote:
>
>> For Wind Speed, global_det wave is using the "WIND" variable. 'UGRD_VGRD'
>> is technically Vector Winds but global_det wave isn't using that.
>>
>> So the changes work, but if statements may be useful to prevent any
>> problems in the future.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2685651387>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BMPTN23ULEYDYXNCLU7OGET2RXXBHAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBVGY2TCMZYG4>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>> [image: malloryprow]*malloryprow* left a comment (NOAA-EMC/EVS#671)
>> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2685651387>
>>
>> For Wind Speed, global_det wave is using the "WIND" variable. 'UGRD_VGRD'
>> is technically Vector Winds but global_det wave isn't using that.
>>
>> So the changes work, but if statements may be useful to prevent any
>> problems in the future.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2685651387>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BMPTN23ULEYDYXNCLU7OGET2RXXBHAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBVGY2TCMZYG4>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

malloryprow commented:
COMOUT is /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/evs/v2.0/plots/global_det/wave.20250226.

### 1. jevs_global_det_wave_grid2obs_plots_last31days
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/plots/global_det/jevs_global_det_wave_grid2obs_plots_last31days_00.o181817051
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_wave_grid2obs_plots_last31days_00.181817051.cbqs01

### 2. jevs_global_det_wave_grid2obs_plots_last90days
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/plots/global_det/jevs_global_det_wave_grid2obs_plots_last90days_00.o181817063
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_wave_grid2obs_plots_last90days_00.181817063.cbqs01

malloryprow commented:
@QiShi-NOAA The jobs are done running. Yesterday in the global meeting you mentioned you updated the website to display the images. Can you put them the website and send a link to it?

QiShi-NOAA commented:
HI mallory,

Will do.

Best,
Qi

On Thu, Feb 27, 2025 at 10:37 AM Mallory Row ***@***.***>
wrote:

> @QiShi-NOAA <https://github.com/QiShi-NOAA> The jobs are done running.
> Yesterday in the global meeting you mentioned you updated the website to
> display the images. Can you put them the website and send a link to it?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688332942>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN22YSSSERYH5OEBFW7L2R4WMZAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBYGMZTEOJUGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: malloryprow]*malloryprow* left a comment (NOAA-EMC/EVS#671)
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688332942>
>
> @QiShi-NOAA <https://github.com/QiShi-NOAA> The jobs are done running.
> Yesterday in the global meeting you mentioned you updated the website to
> display the images. Can you put them the website and send a link to it?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688332942>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN22YSSSERYH5OEBFW7L2R4WMZAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBYGMZTEOJUGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

QiShi-NOAA commented:
The images are displaced on the website:
https://www.emc.ncep.noaa.gov/users/verification/global/gfs/test/wave/grid2obs/wdir/

On Thu, Feb 27, 2025 at 10:37 AM Mallory Row ***@***.***>
wrote:

> @QiShi-NOAA <https://github.com/QiShi-NOAA> The jobs are done running.
> Yesterday in the global meeting you mentioned you updated the website to
> display the images. Can you put them the website and send a link to it?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688332942>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN22YSSSERYH5OEBFW7L2R4WMZAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBYGMZTEOJUGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: malloryprow]*malloryprow* left a comment (NOAA-EMC/EVS#671)
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688332942>
>
> @QiShi-NOAA <https://github.com/QiShi-NOAA> The jobs are done running.
> Yesterday in the global meeting you mentioned you updated the website to
> display the images. Can you put them the website and send a link to it?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688332942>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN22YSSSERYH5OEBFW7L2R4WMZAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBYGMZTEOJUGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

malloryprow commented:
I think we may want to take "Wind Direction" out of the titles and webpage drop down. This would be more consistent with the other plots and webpages for global_det wave.

"Mean Absolute Wind Direction Difference" -> "Mean Absolute Error"
"Mean Wind Direction Difference" -> "Mean Error (Bias)"
"Root Mean Square Wind Direction Difference" -> "Root Mean Square Error"

QiShi-NOAA commented:
Ok, I'll work on that.

On Thu, Feb 27, 2025 at 1:11 PM Mallory Row ***@***.***>
wrote:

> I think we may want to take "Wind Direction" out of the titles and webpage
> drop down. This would be more consistent with the other plots and webpages
> for global_det wave.
>
> "Mean Absolute Wind Direction Difference" -> "Mean Absolute Error"
> "Mean Wind Direction Difference" -> "Mean Error (Bias)"
> "Root Mean Square Wind Direction Difference" -> "Root Mean Square Error"
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688732317>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN226GBNJANEG62HJXQ32R5IN7AVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBYG4ZTEMZRG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: malloryprow]*malloryprow* left a comment (NOAA-EMC/EVS#671)
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688732317>
>
> I think we may want to take "Wind Direction" out of the titles and webpage
> drop down. This would be more consistent with the other plots and webpages
> for global_det wave.
>
> "Mean Absolute Wind Direction Difference" -> "Mean Absolute Error"
> "Mean Wind Direction Difference" -> "Mean Error (Bias)"
> "Root Mean Square Wind Direction Difference" -> "Root Mean Square Error"
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688732317>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN226GBNJANEG62HJXQ32R5IN7AVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBYG4ZTEMZRG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Commit:
Take "Wind Direction" out of the titles

QiShi-NOAA commented:
I've updated the code with "WInd Direction" being removed.from plot title
in the feature branch.

On Thu, Feb 27, 2025 at 1:11 PM Mallory Row ***@***.***>
wrote:

> I think we may want to take "Wind Direction" out of the titles and webpage
> drop down. This would be more consistent with the other plots and webpages
> for global_det wave.
>
> "Mean Absolute Wind Direction Difference" -> "Mean Absolute Error"
> "Mean Wind Direction Difference" -> "Mean Error (Bias)"
> "Root Mean Square Wind Direction Difference" -> "Root Mean Square Error"
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688732317>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN226GBNJANEG62HJXQ32R5IN7AVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBYG4ZTEMZRG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: malloryprow]*malloryprow* left a comment (NOAA-EMC/EVS#671)
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688732317>
>
> I think we may want to take "Wind Direction" out of the titles and webpage
> drop down. This would be more consistent with the other plots and webpages
> for global_det wave.
>
> "Mean Absolute Wind Direction Difference" -> "Mean Absolute Error"
> "Mean Wind Direction Difference" -> "Mean Error (Bias)"
> "Root Mean Square Wind Direction Difference" -> "Root Mean Square Error"
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2688732317>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN226GBNJANEG62HJXQ32R5IN7AVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBYG4ZTEMZRG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

malloryprow commented:
COMOUT is /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/evs/v2.0/plots/global_det/wave.20250226 (cleared out before rerunning).

### 1. jevs_global_det_wave_grid2obs_plots_last31days
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/plots/global_det/jevs_global_det_wave_grid2obs_plots_last31days_00.o181831674
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_wave_grid2obs_plots_last31days_00.181831674.cbqs01

### 2. jevs_global_det_wave_grid2obs_plots_last90days
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/plots/global_det/jevs_global_det_wave_grid2obs_plots_last90days_00.o181831675
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_wave_grid2obs_plots_last90days_00.181831675.cbqs01

QiShi-NOAA commented:
The output looks good and no errors are found in log files.
You can view the images on the website:
https://www.emc.ncep.noaa.gov/users/verification/global/gfs/test/wave/grid2obs/wdir/

On Thu, Feb 27, 2025 at 3:24 PM Mallory Row ***@***.***>
wrote:

> COMOUT is
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/evs/v2.0/plots/global_det/wave.20250226
> (cleared out before rerunning).
> 1. jevs_global_det_wave_grid2obs_plots_last31days
>
> *Log File:*
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/plots/global_det/jevs_global_det_wave_grid2obs_plots_last31days_00.o181831674
> *DATA:*
> /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_wave_grid2obs_plots_last31days_00.181831674.cbqs01
> 2. jevs_global_det_wave_grid2obs_plots_last90days
>
> *Log File:*
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/plots/global_det/jevs_global_det_wave_grid2obs_plots_last90days_00.o181831675
> *DATA:*
> /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_wave_grid2obs_plots_last90days_00.181831675.cbqs01
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2689034864>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN26UVOFIRIHVAWJ2S5L2R5X6FAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBZGAZTIOBWGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: malloryprow]*malloryprow* left a comment (NOAA-EMC/EVS#671)
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2689034864>
>
> COMOUT is
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/evs/v2.0/plots/global_det/wave.20250226
> (cleared out before rerunning).
> 1. jevs_global_det_wave_grid2obs_plots_last31days
>
> *Log File:*
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/plots/global_det/jevs_global_det_wave_grid2obs_plots_last31days_00.o181831674
> *DATA:*
> /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_wave_grid2obs_plots_last31days_00.181831674.cbqs01
> 2. jevs_global_det_wave_grid2obs_plots_last90days
>
> *Log File:*
> /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr671/EVS/dev/drivers/scripts/plots/global_det/jevs_global_det_wave_grid2obs_plots_last90days_00.o181831675
> *DATA:*
> /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_det_wave_grid2obs_plots_last90days_00.181831675.cbqs01
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2689034864>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN26UVOFIRIHVAWJ2S5L2R5X6FAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOBZGAZTIOBWGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

AliciaBentley-NOAA commented:
@QiShi-NOAA Can you please sync your fork? Shannon's subseasonal PR was merged earlier today. Also, does anything else need to be rerun on this PR before it is merged? 

CC @PerryShafran-NOAA

QiShi-NOAA committed:
Merge branch 'NOAA-EMC:develop' into wind_direction

QiShi-NOAA commented:
HI Alicia,

I've synced my wind direction fork. No rerun needed on this PR before it is
merged.

Best,
Qi

On Mon, Mar 3, 2025 at 8:57 PM Alicia Bentley ***@***.***>
wrote:

> @QiShi-NOAA <https://github.com/QiShi-NOAA> Can you please sync your
> fork? Shannon's subseasonal PR was merged earlier today. Also, does
> anything else need to be rerun on this PR before it is merged?
>
> CC @PerryShafran-NOAA <https://github.com/PerryShafran-NOAA>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2695983647>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN24TIEGWQ7E24ZOZLO32SUB6RAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOJVHE4DGNRUG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: AliciaBentley-NOAA]*AliciaBentley-NOAA* left a comment
> (NOAA-EMC/EVS#671)
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2695983647>
>
> @QiShi-NOAA <https://github.com/QiShi-NOAA> Can you please sync your
> fork? Shannon's subseasonal PR was merged earlier today. Also, does
> anything else need to be rerun on this PR before it is merged?
>
> CC @PerryShafran-NOAA <https://github.com/PerryShafran-NOAA>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/671#issuecomment-2695983647>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMPTN24TIEGWQ7E24ZOZLO32SUB6RAVCNFSM6AAAAABX5R4PBSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOJVHE4DGNRUG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

PerryShafran-NOAA commented:
@QiShi-NOAA I'll be doing the final code review and then I'll merge this code.

PerryShafran-NOAA reviewed (approved):
Code changes work as expected.  Approved for merge.
